src/os/bluestore/BlueFS.cc

```diff
@@ -3799,7 +3803,9 @@ int BlueFS::_allocate(uint8_t id, uint64_t len,
     }   
     ++alloc_attempts;
     extents.reserve(4);  // 4 should be (more than) enough for most allocations
+    auto start = mono_clock::now();
     alloc_len = alloc[id]->allocate(need, alloc_unit, hint, &extents);
+    logger->tinc(l_bluefs_allocator_lat, mono_clock::now() - start);
```

Comment:
IMO it would be better to have separate perf counters for each device type: main, db, wal.
This will provide a bit more insight on bluefs operationing.

Reply:
i got it.

Reply:
change show:
```
# ceph -c ceph.conf daemon ./asok/osd.0.asok perf dump
...
    "bluefs": {
...
        "wal_alloc_lat": {
            "avgcount": 3,
            "sum": 0.000036683,
            "avgtime": 0.000012227
        },
        "db_alloc_lat": {
            "avgcount": 11,
            "sum": 0.000145876,
            "avgtime": 0.000013261
        },
        "db_fallback_bdev_alloc_lat": {
            "avgcount": 0,
            "sum": 0.000000000,
            "avgtime": 0.000000000
        }
...
    "bluestore": {
...
        "allocator_lat": {
            "avgcount": 649,
            "sum": 0.009117295,
            "avgtime": 0.000014048
        }
...
```